.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,11 @@
+name: pre-commit
+
+on: [push]
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+    - uses: pre-commit/action@v2.0.0
```

Comment:
This links to the `.pre-commit.yaml` file below?

Reply:
yes

Reply:
Maybe we make CCV template repository at a later date?

Reply:
For precommit templates per language